homeassistant/components/hassio/__init__.py

```diff
@@ -158,6 +158,17 @@ async def async_uninstall_addon(hass: HomeAssistantType, slug: str) -> None:
     await hassio.send_command(command)
 
 
+@bind_hass
+async def async_start_addon(hass: HomeAssistantType, slug: str) -> None:
+    """Start add-on.
+
+    The caller of the function should handle HassioAPIError.
+    """
+    hassio = hass.data[DOMAIN]
+    command = f"/addons/{slug}/start"
+    await hassio.send_command(command)
```

Comment:
Should this validate the configuration before trying to start it?

Reply:
I guess it does not matter since the user will not be aware that this is happening anyway

Reply:
Yeah, maybe that's a good idea. I can add that.

Reply:
Right. Ok, let's skip it then.